x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/TransportResumeFollowAction.java

```diff
@@ -515,7 +515,8 @@ static String[] extractLeaderShardHistoryUUIDs(Map<String, String> ccrIndexMetad
         DataTier.TIER_PREFERENCE_SETTING,
         IndexSettings.BLOOM_FILTER_ID_FIELD_ENABLED_SETTING,
         IndexSettings.LIFECYCLE_ORIGINATION_DATE_SETTING,
-        IndexSettings.LIFECYCLE_PARSE_ORIGINATION_DATE_SETTING
+        IndexSettings.LIFECYCLE_PARSE_ORIGINATION_DATE_SETTING,
+        IndexSettings.PREFER_ILM_SETTING
```

Comment:
Do we all agree this should not be replicated?

My reasoning is that the leader and follower clusters could have different tiering strategies and use different management systems

Reply:
We normally don't replicate settings when they can "double-up" operations. I think we should replicate the setting for now, similar to how we replicate the `index.lifecycle.name` setting